.github/workflows/ci.yml

```diff
@@ -176,7 +176,8 @@ jobs:
           generator-tag: v6.6.0
           command-args: |
             --output $GENERATOR_OUTPUT_DIR_CI \
-            --ignore-file-override $GENERATOR_IGNORE_FILE
+            --ignore-file-override $GENERATOR_IGNORE_FILE \
+            --additional-properties=useSingleRequestParameter=true
```

Comment:
The affect of this setting is that the generated API client puts all API parameters as an object in the first argument, rather than as position parameters. I.e.,
```ts
// now we get
apiClient. coursesList({ limit, offset })
// not
apiClient.courseList(limit, offset)
``` 

My thoughts are:
- position parameters will be annoying to deal with if we change the order
- position parameters are annoying when we want to pass 2nd param but not 1st.
- having all API params in the first argument is a little more convenient when dealing with Typescript, since OpenAPI Generator makes interfaces for the type signatures. *Although the interface names are extremely verbose, like `LearningResourcesApiLearningResourcesListRequest`*